subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/logging/TestWorkerProgressListener.java

```diff
@@ -32,6 +32,8 @@
 
 public class TestWorkerProgressListener implements TestListenerInternal {
 
+    private static final int MAX_TEST_NAME_LENGTH = 60;
```

Comment:
:o: I would have probably just hard-coded this in `JavaClassNameFormatter` for now.